### DISCUSSION TITLE
The bulb can not be connected via WiZ Light integration.

### DISCUSSION THREAD

**Discussion by thomasjedstrom** [sbidy/wiz_light#194] [Q&A]
Greetings!

First off much appreciation for the work you all have done on this.  I've run into an issue where I was able to correctly pair one WiZ bulb I bought from Home Depot but not the other and I'm not sure why.

![IMG_8226](https://user-images.githubusercontent.com/13879165/139604180-d9bcb668-ebdd-48f1-bf28-c8752d483c35.jpg)

For the bulb on the right, I used its IP address as the host name and it correctly found the device and created an entity for it.

For the bulb on the left, if I use the IP address, it adds the integration but no device/entity is created found.  If I use the `wiz-{last-6-digits-of-MAC-address}`, I get the following error:

![image](https://user-images.githubusercontent.com/13879165/139604224-8eb4f49f-3fc8-4827-814b-152413f65c1e.png)

Any ideas on what's going on?

Just to round out exactly what happens, using the ip address, it goes to this:

![image](https://user-images.githubusercontent.com/13879165/139604260-229cff22-4a57-4167-8edd-f8f28e326671.png)

at which point the integration tile looks like this:

![image](https://user-images.githubusercontent.com/13879165/139604264-b03c6124-127c-4a91-904e-4e373d370f2c.png)

and the bulb on the right (the `Office Overhead Light`) shows 1 device and 1 entity correctly:

![image](https://user-images.githubusercontent.com/13879165/139604285-0c7251cd-bd85-49a0-a5ea-ae7a40961fe0.png)

but the bulb on the left looks like this:

![image](https://user-images.githubusercontent.com/13879165/139604294-f9ed6277-fdeb-4feb-86b5-8d5986dd0f87.png)

Appreciate any help on this.  I do have them both configured in my `configuration.yaml`, although I'm not sure what effect that actually had since I still had to add the integration through the UI.  Anyways, appreciate you all.

**Comment by gomonte** [comment]
Hi there, I'm not really an expert but I just wanted to point some common issues not really for these devices but in general that could maybe help you.

Can you control both bulbs with the Wiz app?
Have you checked that IPs are static? (check if you don't have a repeated IP maybe)
If you can change IPs, can you exchange devices IPs just to see what happens...

Also not sure if it is rigth to add bulbs by the integration and by the configuration.yaml, have you tried just using one method at a time?

While waiting for some more elaborated help maybe you can post the part of your configuration.yaml showing these devices and the logs regarding them.

Hope somebody can help you better. I'm just waiting for three Wiz bulbs (GU10 spots) for my bathroom and I hope they will work without any problem thanks to the guys developping this integration.
